src/sherpa_ai/prompts/prompt_loader.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict, List, Optional, Union
+import json
+
+from sherpa_ai.utils import JsonToObject, load_json
+# from sherpa_ai.prompts.json_to_object import JsonToObject
+
+class PromptLoader:
+    def __init__(self, json_file_path: str):
+        raw_data = load_json(json_file_path)
+        # Convert JSON to object
+        self.data = JsonToObject()(raw_data)
+        self.prompts = self._process_prompts()
+    
+    def _process_prompts(self) -> Dict:
+        """
+        Process the raw prompts data into a structured format.
+        
+        Returns:
+            Dict: Processed prompts organized by wrapper
+        """
+        processed_prompts = {}
+        for wrapper_key, wrapper_data in self.data.__dict__.items():
```

Comment:
If we convert a dictionary into an object and use `__dict__` of that object, then we are doing something wrong!